### PR TITLE
[Docs] Add `how to run`

### PR DESCRIPTION
**Pull request by RTAndrew**

### CODE REVIEW

File: lib/locales/br/frontend/common.json [hunk 1]
```diff
@@ -340,5 +343,16 @@
...
+    "comment": "Comentar",
+    "comments": "Comentários",
+    "quote_reply": "Responder citando",
+    "leave_a_comment": "Deixe um comentário...",
+    "created_successfully": "O comentário foi criado com sucesso",
+    "delete": "Excluir comentário",
+    "delete_description": "Excluir este comentário é irreversível. <1>Tem certeza de que deseja continuar?</1>",
+    "deleted_successfully": "O comentário foi excluído com sucesso"
+  },
+  "select_voting_track": "Selecionar track de votação"
```
Comment (JoseMarshall) [inline lib/locales/br/frontend/common.json:357]: This is already in this PR:
https://github.com/layerx-labs/taikai-locales/pull/6/files#diff-9f4990d63a64dad3d5417ea9222791b1764ed4c8a21859f18a374c7f31319b55

File: lib/types/enums.ts [hunk 2]
```diff
@@ -1285,9 +1285,14 @@ export enum i18nKey {
   COMMON__GALLERY = "common:gallery",
   COMMON__SEARCH = "common:search",
   COMMON__VIEWS = "common:views",
+  COMMON__VIDEO = "common:video",
+  COMMON__COVER = "common:cover",
+  COMMON__AVATAR = "common:avatar",
```
Comment (JoseMarshall) [inline lib/types/enums.ts:1290]: Same here:
https://github.com/layerx-labs/taikai-locales/pull/6/files#diff-b83b60c5d2f215a908e4cda192abf74ab8948a92cac764cedee1578b5d3c863a

File: README.md [hunk 3]
```diff
@@ -101,6 +101,32 @@ export type Locale = 'en' | 'br' | '<MY_NEW_LOCALE>';
...
+## How To Run
+### Locally
+
+To visualize your changes in another repository/project, execute the following commands:
+```
+npm run build
+```
+```
+npm pack
+```
```
Comment (JoseMarshall) [inline README.md:113]: You don't need to tun `npm pack` the build script already do that, there is a `postbuild` hook indeed.

File: README.md [hunk 4]
```diff
@@ -101,6 +101,32 @@ export type Locale = 'en' | 'br' | '<MY_NEW_LOCALE>';
...
+After that, it will create file with a name `taikai-locales[version].tgz` in the root of your project. Copy its relative path, then proceed to another project and execute:
+```
+npm i /Users/[path-to-project]/taikai-locales[version].tgz
+```
```
Comment (JoseMarshall) [inline README.md:117]: nice, I like it!!

**Review by JoseMarshall — CHANGES_REQUESTED** [review]
Remove the commits that brings changes from this PR: 
https://github.com/layerx-labs/taikai-locales/pull/6

And check the comment about the `npm pack` when running locally.

File: README.md (hunk 3 above)
Comment (RTAndrew) [reply README.md:113]: Thanks. Did not pay attention to it

**Review by JoseMarshall — APPROVED** [review]

**Pull request merged**